setup.py

```diff
@@ -0,0 +1,58 @@
+import re
+import ast
+
+from setuptools import find_packages, setup
+
+# https://github.com/mitsuhiko/flask/blob/master/setup.py
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+
+with open('americangut/__init__.py', 'rb') as f:
+    hit = _version_re.search(f.read().decode('utf-8')).group(1)
+    version = str(ast.literal_eval(hit))
+
+classes = """
+    Development Status :: 1 - Planning
+    Topic :: Software Development :: Libraries
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Operating System :: Unix
+    Operating System :: POSIX
+    Operating System :: MacOS :: MacOS X
+"""
+classifiers = [s.strip() for s in classes.split('\n') if s]
+
+setup(name='americangut',
+      version=version,
+      description="American gut analyses",
+      long_description="American Gut open-access data "
+                       "and IPython notebooks",
+      author="American Gut development team",
+      author_email="jamietmorton@gmail.com",
```

Comment:
...actually, Just use "American Gut development team" and the American Gut help account email.